README.md

```diff
@@ -322,6 +323,37 @@ cv2.imwrite('youtube_frame.jpg', frame)
 grabber.release()
 ```
 
+### File Stream
+Here's an example of using FrameGrab to capture frames from a video file:
+
+```python
+from framegrab import FrameGrabber
+import cv2
+
+config = {
+    'input_type': 'file_stream',
+    'id': {
+        'file_path': 'path/to/your/video.mjpeg'  # or .mp4, .avi, etc.
+    },
+    'options': {
+        'max_fps': 2,  # Decimate stream if FPS is too high
```

Comment:
This comment is perhaps confusing to end users. Maybe something like "specify a lower FPS than the original video's FPS if desired. Framegrab will skip extra frames as needed."

I also wonder if we should just call this "fps" instead of "max_fps". I was the one who introduced the term "max_fps" for RTSP streams, and it is admittedly confusing. It determines the rate at which the frames are grabbed from the network stream, effectively emptying the buffer. As long as this value is greater than or equal to the rate frames are actually being published by the camera, the buffer won't grow, hence "max_fps". 

The way you are using it is a bit different. You can effectively lower the FPS of the camera if you want to. If you tried the same thing with an RTSP grabber, it would cause problems with the buffer.

Reply:
hmmm, its still a `max_fps` though - we cant speed up the video's underlying FPS and so this can only cap the FPS.